sdk/eventhub/Azure.Messaging.EventHubs/tests/Infrastructure/EventHubScope.cs

```diff
@@ -104,19 +104,6 @@ public async ValueTask DisposeAsync()
         public static Task<EventHubScope> CreateAsync(int partitionCount,
                                                       [CallerMemberName] string caller = "") => CreateAsync(partitionCount, Enumerable.Empty<string>(), caller);
 
-        /// <summary>
-        ///   Performs the tasks needed to create a new Event Hub instance with the requested
-        ///   partition count and a dynamically assigned unique name.
-        /// </summary>
-        ///
-        /// <param name="partitionCount">The number of partitions that the Event Hub should be configured with.</param>
-        /// <param name="consumerGroup">The name of a consumer group to create and associate with the Event Hub; the default consumer group should not be specified, as it is implicitly created.</param>
-        /// <param name="caller">The name of the calling method; this is intended to be populated by the runtime.</param>
-        ///
-        public static Task<EventHubScope> CreateAsync(int partitionCount,
```

Comment:
This overload couldn't be used without either specifying the parameter name when calling or providing the `caller`  value meant to be injected by the runtime.  It was seldom used and was the cause for at least one bug.   Removing it in favor of the overload that accepts an enumerable.